.github/workflows/pytest.yaml

```diff
@@ -71,12 +71,9 @@ jobs:
       run: python -m pip install --upgrade pip setuptools wheel
       shell: bash
     - name: install qcodes with minimum requirements
-#     Use fork of requirements-builder until official version
-#     has support for pyproject.toml (pep621)
       run: |
-        pip install git+https://github.com/jenshnielsen/requirements-builder.git
-        pip install versioningit
-        requirements-builder -l min -e test setup.py -o min-requirements.txt
```

Comment:
Does not seem to work correct at the moment

Reply:
hmm looks exactly how it's described in the docs. could it be the reference to test dependencies in requirments.in?

Reply:
@trevormorgan This should now be working by compiling the dependencies directly from pyproject.toml